packages/contracts/contracts/test/L2OutputOracle.t.sol

```diff
@@ -565,3 +565,52 @@ contract L2OutputOracleUpgradeable_Test is L2OutputOracle_Initializer {
         assertEq(slot21Expected, slot21After);
     }
 }
+
+contract L2OutputOracle_Restricted_Test is ValidatorSystemUpgrade_Initializer {
+    function setUp() public virtual override {
+        super.setUp();
+
+        vm.prank(trusted);
+        pool.deposit{ value: trusted.balance }();
+        _registerValidator(trusted, minActivateAmount);
+
+        // Submit outputs to ValidatorPool is terminated
+        for (uint256 i; i <= terminateOutputIndex; i++) {
+            _submitL2OutputV1();
+        }
```

Comment:
Since the first output of V2 is always for trusted validator, so for accurate test, we should pass the first V2 output
```suggestion
        }

        _submitL2OutputV2(false);
```